AppInspector/rules/default/tools/pipeline.json

```diff
@@ -0,0 +1,325 @@
+[
+  {
+    "name": "Pipeline Tool (CredScan)",
+    "id": "AI053300",
+    "description": "Pipeline Tool - CredScan",
+    "tags": [
+      "Metadata.Pipeline.Tools.CredScan"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "CredScan",
+        "type": "string",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (PoliCheck)",
+    "id": "AI053400",
+    "description": "Authentication using Microsoft Identity Platform (ADAL/MSAL) or similar",
+    "tags": [
+      "Metadata.Pipeline.Tools.PoliCheck"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "PoliCheck@1",
+        "type": "string",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (AntiMalware)",
+    "id": "AI053500",
+    "description": "Authentication using Microsoft Identity Platform (ADAL/MSAL) or similar",
+    "tags": [
+      "Metadata.Pipeline.Tools.AntiMalware"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "AntiMalware",
+        "type": "string",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (CodeSigning)",
+    "id": "AI053600",
+    "description": "Authentication using Microsoft Identity Platform (ADAL/MSAL) or similar",
+    "tags": [
+      "Metadata.Pipeline.Tools.CodeSigning"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "EsrpCodeSigning",
+        "type": "string",
+        "scopes": [ "code", "comment" ],
+        "modifiers": [ "i" ],
+        "confidence": "high",
+        "_comment": ""
+      }
+    ]
+  },
+  {
+    "name": "Pipeline Tool (RoslynAnalyzers)",
+    "id": "AI053700",
+    "description": "Roslyn",
+    "tags": [
+      "Metadata.Pipeline.Tools.RoslynAnalyzers"
+    ],
+    "severity": "critical",
+    "patterns": [
+      {
+        "pattern": "RoslynAnalyzers@2",
```

Comment:
Suggest just `RoslynAnalyzers@` since the version could bump, or be `@1`, I suppose.